ios/FluentUI/Notification/NotificationTokens.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Pre-defined styles of the notification
+@objc public enum MSFNotificationStyle: Int, CaseIterable {
+    case primaryToast
+    case neutralToast
+    case primaryBar
+    case primaryOutlineBar
+    case neutralBar
+    case dangerToast
+    case warningToast
+
+    var isToast: Bool {
+        switch self {
+        case .primaryToast,
+             .neutralToast,
+             .dangerToast,
+             .warningToast:
+            return true
+        case .primaryBar,
+             .primaryOutlineBar,
+             .neutralBar:
+            return false
+        }
+    }
+
+    var animationDurationForShow: TimeInterval { return isToast ? Constants.animationDurationForShowToast : Constants.animationDurationForShowBar }
+    var animationDurationForHide: TimeInterval { return Constants.animationDurationForHide }
+    var animationDampingRatio: CGFloat { return isToast ? Constants.animationDampingRatioForToast : 1 }
+
+    var needsFullWidth: Bool { return !isToast }
+    var needsSeparator: Bool { return  self == .primaryOutlineBar }
+    var supportsTitle: Bool { return isToast }
+    var supportsImage: Bool { return isToast }
+    var shouldAlwaysShowActionButton: Bool { return isToast }
+
+    private struct Constants {
+        static let animationDurationForShowToast: TimeInterval = 0.6
+        static let animationDurationForShowBar: TimeInterval = 0.3
+        static let animationDurationForHide: TimeInterval = 0.25
+        static let animationDampingRatioForToast: CGFloat = 0.5
+    }
+}
+
+public class NotificationTokens: ControlTokens {
+    public internal(set) var style: MSFNotificationStyle = .primaryToast
```

Comment:
Could you provide quick one-line comments for each of the `public` and `open` vars here?

I know not all of the converted token classes have them, but we're definitely moving that direction. In the meantime check out `ActivityIndicatorTokens` for an example.